front/cypress/integration/project_card.ts

```diff
@@ -30,20 +31,17 @@ describe('Project card component', () => {
   });
 
   it('shows the title, description, progress bar and cta', () => {
-    cy.get('.e2e-project-card')
-      .contains(projectTitle)
-      .closest('.e2e-project-card')
-      .as('projectCard');
-    cy.get('@projectCard')
-      .get('.e2e-project-card-project-title')
-      .contains(projectTitle);
-    cy.get('@projectCard')
```

Comment:
This test was super slow, failing once for no reason and then passing in the retry. This is a lot faster